Altis_Life.Altis/core/utils/fn_sideToString.sqf

```diff
@@ -0,0 +1,26 @@
+/*
+    File: fn_sideToString.sqf
+    Author: DomT602
+    Description:
+    Translates a side to a string representing the side.
+    To-do - add to functions.hpp, add to spyglass, move into utils folder
+*/
+params [
+    ["_unit",objNull,[objNull]],
+    ["_lowerCase",false,[true]]
+];
+if (isNull _unit) exitWith {};
+
+private _sideText = switch (side _unit) do {
```

Comment:
I think we should use if-call-exitWith instead of if-switch:
https://community.bistudio.com/wiki/Code_Optimisation#if_and_switch

Reply:
The performance difference mainly applies to when switch (true) is used, this string setup works fine.